### PR TITLE
App settings applications service backend and Key Vault

### PR DESCRIPTION
**Pull request by andylangridge**
- Added a Key Vault to the common stack, and all the required secrets are also created in this stack with placeholder values.
- List of all Secret URIs are then output as a variable including the Microsoft reference string format so they can be consumed by other stacks
- Updated CD pipeline to deploy individual stacks
- Added App Settings for the Applications Service Backend web app

### CODE REVIEW

File: app/stacks/applications-service/variables/dev.hcl [hunk 1]
```diff
@@ -0,0 +1,3 @@
+locals {
+  mysql_database = "ipclive"
```
Comment (lestermarch) [inline app/stacks/applications-service/variables/dev.hcl:2]: Should this be `ipcdev`?

File: app/stacks/applications-service/variables/test.hcl [hunk 2]
```diff
@@ -0,0 +1,3 @@
+locals {
+  mysql_database = "ipclive"
```
Comment (lestermarch) [inline app/stacks/applications-service/variables/test.hcl:2]: Should this be `ipctest`?

File: app/stacks/applications-service/variables/dev.hcl (hunk 1 above)
Comment (andylangridge) [reply app/stacks/applications-service/variables/dev.hcl:2]: That's the current value in the existing dev!

**Review by lestermarch — APPROVED** [review]

File: app/stacks/applications-service/variables/dev.hcl (hunk 1 above)
Comment (andylangridge) [reply app/stacks/applications-service/variables/dev.hcl:2]: Will be finding out the real values soon. So for now these are ok, especially seeing as we have no way to connect to the actual database!

**Pull request merged**